Framework/TestWorkflows/src/dataSamplingParallel.cxx

```diff
@@ -143,7 +143,11 @@ WorkflowSpec defineDataProcessing(ConfigContext const&)
   specs.push_back(simpleQcTask);
   specs.push_back(dummyProducer);
 
-  std::string configurationSource = std::string("json:/") + getenv("O2_ROOT") + "/share/etc/exampleDataSamplingConfig.json";
+  const char* o2Root = getenv("O2_ROOT");
+  if (o2Root == nullptr) {
```

Comment:
Just a (stupid?) idea here : would it be possible and/or desirable to try to deduce the location of the share directory from the executable path (e.g assuming the exe is in bin the share is in ../share or something alike) ? (the use case would not be limited to this case but in all places we use O2_ROOT in our code) ?

Reply:
Even better: we should have a "FileInPath" utility in the framework which does the lookup for you, so that the user does not need to worry about it (with relative to executable being the default).

Reply:
Now I would leave it as it is, but indeed having such an utility would help.